### PR TITLE
Packages: add Autoloader suggestion

### PR DESCRIPTION
**Pull request by jeherve**
## Proposed changes:

This should help package consumers who may be interested in using one of those packages in one of their plugins. In that situation, using the autoloader is a good idea, to avoid any conflicts on sites that use multiple plugins relying on those packages.

This PR does 3 things:

1. It adds a mention of the Autoloader at the bottom of all our packages' readme (all the packages that do get shipped with plugins)
2. It suggests the use of the autoloader for those packages using Composer's `suggest`: https://getcomposer.org/doc/04-schema.md#suggest
3. It automates the 2 things above for future packages by adding them to the `jetpack generate` command.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

* p1680719718703189-slack-G04PEFM2P60

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Not much to test for here; look out for typos.

### CODE REVIEW

**Comment by jeherve** [comment]
Noting that the failed gardening check is expected; there are too many labels in this PR:

```
Error: main: Task addLabels failed with error: HttpError: Validation Failed. Issues cannot have more than 100 labels
```

**Review by zinigor — DISMISSED** [review]
Thank you for working on this!

File: projects/packages/backup/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+If you plan on using this package in your WordPress plugin, we would recommend that you use [Jetpack Autoloader](https://packagist.org/packages/automattic/jetpack-autoloader) as your autoloader. This will allow for maximum interoperability with other plugins that use this package as well.
+
+## Security
+
+Need to report a security vulnerability? Go to [https://automattic.com/security/](https://automattic.com/security/) or directly to our security bug bounty site [https://hackerone.com/automattic](https://hackerone.com/automattic).
+
+## License
+
+jetpack-backups is licensed under [GNU General Public License v2 (or later)](./LICENSE.txt)
```
Comment (darssen) [inline projects/packages/backup/README.md]:
```suggestion
jetpack-backup is licensed under [GNU General Public License v2 (or later)](./LICENSE.txt)
```

File: projects/packages/a8c-mc-stats/composer.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/a8c-mc-stats/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/abtest/composer.json [hunk 3]
```diff
@@ -12,6 +12,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/abtest/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/action-bar/composer.json [hunk 4]
```diff
@@ -12,6 +12,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/action-bar/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/admin-ui/composer.json [hunk 5]
```diff
@@ -10,6 +10,9 @@
 		"automattic/jetpack-logo": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/admin-ui/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/assets/composer.json [hunk 6]
```diff
@@ -12,6 +12,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"wikimedia/testing-access-wrapper": "^1.0 || ^2.0"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/assets/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/blaze/composer.json [hunk 7]
```diff
@@ -13,6 +13,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/blaze/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/blocks/composer.json [hunk 8]
```diff
@@ -10,6 +10,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/blocks/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/compat/composer.json [hunk 9]
```diff
@@ -7,6 +7,9 @@
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/compat/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/config/composer.json [hunk 10]
```diff
@@ -7,6 +7,9 @@
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/config/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/connection/composer.json [hunk 11]
```diff
@@ -17,6 +17,9 @@
 		"brain/monkey": "2.6.1",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/connection/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/constants/composer.json [hunk 12]
```diff
@@ -9,6 +9,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/constants/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/device-detection/composer.json [hunk 13]
```diff
@@ -8,6 +8,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/device-detection/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/error/composer.json [hunk 14]
```diff
@@ -8,6 +8,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/error/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/forms/composer.json [hunk 15]
```diff
@@ -15,6 +15,9 @@
 		"automattic/jetpack-connection": "@dev",
 		"automattic/wordbless": "^0.4.1"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/forms/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/google-fonts-provider/composer.json [hunk 16]
```diff
@@ -9,6 +9,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"brain/monkey": "2.6.1"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/google-fonts-provider/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/heartbeat/composer.json [hunk 17]
```diff
@@ -9,6 +9,9 @@
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/heartbeat/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/identity-crisis/composer.json [hunk 18]
```diff
@@ -15,6 +15,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/identity-crisis/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/image-cdn/composer.json [hunk 19]
```diff
@@ -12,6 +12,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/image-cdn/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/import/composer.json [hunk 20]
```diff
@@ -11,6 +11,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/import/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/ip/composer.json [hunk 21]
```diff
@@ -9,6 +9,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/ip/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/jetpack-mu-wpcom/composer.json [hunk 22]
```diff
@@ -9,6 +9,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "0.4.2"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/jetpack-mu-wpcom/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/jitm/composer.json [hunk 23]
```diff
@@ -18,6 +18,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/jitm/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/lazy-images/composer.json [hunk 24]
```diff
@@ -12,6 +12,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/lazy-images/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/licensing/composer.json [hunk 25]
```diff
@@ -11,6 +11,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/licensing/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/logo/composer.json [hunk 26]
```diff
@@ -8,6 +8,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/logo/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/my-jetpack/composer.json [hunk 27]
```diff
@@ -19,6 +19,9 @@
 		"automattic/wordbless": "@dev",
 		"automattic/jetpack-videopress": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/my-jetpack/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/options/composer.json [hunk 28]
```diff
@@ -10,6 +10,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/options/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/partner/composer.json [hunk 29]
```diff
@@ -13,6 +13,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/partner/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: tools/cli/commands/generate.js [hunk 30]
```diff
@@ -556,6 +556,9 @@ async function createComposerJson( composerJson, answers ) {
 				'::PACKAGE_VERSION': `src/class-${ answers.name }.php`,
 			};
 			composerJson.type = 'jetpack-library';
+			composerJson.suggest ||= {};
+			composerJson.suggest[ 'automattic/jetpack-autoloader' ] =
+				'Allow for better interopability with other plugins that use this package.';
```
Comment (darssen) [inline tools/cli/commands/generate.js]:
```suggestion
				'Allow for better interoperability with other plugins that use this package.';
```

File: projects/packages/yoast-promo/composer.json [hunk 31]
```diff
@@ -8,6 +8,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/yoast-promo/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/wp-js-data-sync/composer.json [hunk 32]
```diff
@@ -9,6 +9,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/wp-js-data-sync/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/wordads/composer.json [hunk 33]
```diff
@@ -14,6 +14,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/wordads/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/waf/composer.json [hunk 34]
```diff
@@ -15,6 +15,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/waf/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/videopress/composer.json [hunk 35]
```diff
@@ -15,6 +15,9 @@
 		"automattic/wordbless": "@dev",
 		"brain/monkey": "2.6.1"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/videopress/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/transport-helper/composer.json [hunk 36]
```diff
@@ -11,6 +11,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/transport-helper/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/tracking/composer.json [hunk 37]
```diff
@@ -11,6 +11,9 @@
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/tracking/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/sync/composer.json [hunk 38]
```diff
@@ -17,6 +17,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/sync/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/status/composer.json [hunk 39]
```diff
@@ -12,6 +12,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/jetpack-ip": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/status/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/stats/composer.json [hunk 40]
```diff
@@ -14,6 +14,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/stats/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/stats-admin/composer.json [hunk 41]
```diff
@@ -15,6 +15,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "dev-master"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/stats-admin/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/search/composer.json [hunk 42]
```diff
@@ -16,6 +16,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/wordbless": "0.4.2"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/search/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/roles/composer.json [hunk 43]
```diff
@@ -9,6 +9,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/roles/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/redirect/composer.json [hunk 44]
```diff
@@ -11,6 +11,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/redirect/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/publicize/composer.json [hunk 45]
```diff
@@ -16,6 +16,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/wordbless": "0.4.2"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/publicize/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/password-checker/composer.json [hunk 46]
```diff
@@ -9,6 +9,9 @@
 		"automattic/wordbless": "@dev",
 		"yoast/phpunit-polyfills": "1.0.4"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/password-checker/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/plans/composer.json [hunk 47]
```diff
@@ -12,6 +12,9 @@
 		"automattic/jetpack-status": "@dev",
 		"automattic/wordbless": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/plans/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/plugin-deactivation/composer.json [hunk 48]
```diff
@@ -10,6 +10,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/plugin-deactivation/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/plugins-installer/composer.json [hunk 49]
```diff
@@ -10,6 +10,9 @@
 		"yoast/phpunit-polyfills": "1.0.4",
 		"automattic/jetpack-changelogger": "@dev"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/plugins-installer/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

File: projects/packages/post-list/composer.json [hunk 50]
```diff
@@ -11,6 +11,9 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"yoast/phpunit-polyfills": "1.0.4"
 	},
+	"suggest": {
+		"automattic/jetpack-autoloader": "Allow for better interopability with other plugins that use this package."
```
Comment (darssen) [inline projects/packages/post-list/composer.json]:
```suggestion
		"automattic/jetpack-autoloader": "Allow for better interoperability with other plugins that use this package."
```

**Review by darssen — COMMENTED** [review]
Added suggestions for  interopability typos and renaming jetpack-backups to jetpack-backup

**Review by jeherve — COMMENTED** [review]
@darssen Thank you for the review! 🙇‍♂️ Could I get another one now, so this can be merged? :)

**Review by darssen — APPROVED** [review]
LGTM!

**Pull request merged**